src/CMakeLists.txt

```
@@ -135,6 +137,18 @@ if(ENABLE_SIO)
   LIST(APPEND INSTALL_LIBRARIES podioSioIO podioSioIODict)
 endif()
 
+# --- IO
+set(io_sources
+  Writer.cc
+  Reader.cc
+  )
+
+PODIO_ADD_LIB_AND_DICT(podioIO "${core_headers}" "${io_sources}" io_selection.xml)
+target_link_libraries(podioIO PUBLIC podio::podio podio::podioRootIO)
+if(ENABLE_SIO)
+  target_link_libraries(podioIO PUBLIC podio::podioSioIO)
+endif()
+
 # --- Install everything
 install(TARGETS podio podioDict podioRootIO podioRootIODict ${INSTALL_LIBRARIES}
```

```suggestion
install(TARGETS podio podioDict podioRootIO podioRootIODict podioIO ${INSTALL_LIBRARIES}
```

I don't think I saw it anywhere else that would install it(?).

---

Fixed, I checked the target, dictionaries and headers are being installed